pkgs/tools/misc/djent/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, mpfr
+}:
+
+stdenv.mkDerivation rec {
+  pname = "djent";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "dj-on-github";
+    repo = "djent";
+    rev = "${version}";
+    hash = "sha256-inMh7l/6LlrVnIin+L+fj+4Lchk0Xvt09ngVrCuvphE=";
+  };
+
+  buildInputs = [ mpfr ];
+
+  preBuild = lib.optional stdenv.hostPlatform.isAarch64 ''
+    sed -i s/gcc/${stdenv.cc.targetPrefix}gcc/g Makefile
+    sed -i s/-m64//g Makefile
+  '';
+
+  installPhase = ''
+    install -D djent $out/bin/djent
+  '';
+
+  meta = {
+    homepage = "http://www.deadhat.com/";
+    description = ''A reimplementation of the Fourmilab/John Walker random number test program
+      ent with several improvements'';
+    license = lib.licenses.gpl2Only;
+    platforms = lib.platforms.linux;
```

Comment:
does it work on *bsd?

Reply:
Now it does work on darwin (https://github.com/NixOS/nixpkgs/pull/245719/checks?check_run_id=16207102225) and also when cross-compiling locally for x86_64 netbsd and freebsd.